lib/x11utils.pm

```diff
@@ -112,6 +120,8 @@ sub ensure_unlocked_desktop {
                     assert_screen 'generic-desktop';
                 }
                 else {
+                    diag("Next loop ($counter), Generic desktop didn't match");
+                    record_soft_failure('bsc#1168979') if is_aarch64;
```

Comment:
If you remove that `record_soft_failure`, I can approve.
This will record a soft fail anytime the desktop runner is not shown, even when alt-f2 keys a missing.